Fenced_Frames_Ads_Reporting.md

```diff
@@ -47,7 +47,7 @@ The following new APIs will be added for achieving this.
 
 ## reportEvent (preregistered destination URL)
 
-There are two sets of parameters that can be used to invoke the `reportEvent` API. In the first, fenced frames can invoke the `reportEvent` API to tell the browser to send a beacon with event data to a URL registered by the worklet in `registerAdBeacon` (see below). Depending on the declared `destination`, the beacon is sent to either the buyer's or the seller's registered URL. Examples of such events are mouse hovers, clicks (which may or may not lead to navigation e.g. video player control element clicks), etc.
+There are two variants of the `reportEvent` API that are invoked with different sets of parameters. In the first variant, fenced frames can invoke the `reportEvent` API to tell the browser to send a beacon with event data to a URL registered by the worklet in `registerAdBeacon` (see below). Depending on the declared `destination`, the beacon is sent to either the buyer's or the seller's registered URL. Examples of such events are mouse hovers, clicks (which may or may not lead to navigation e.g. video player control element clicks), etc.
```

Comment:
there is a third variant for private aggregation.

"In addition to reportEvent being used for private aggregation, as described in <explainer link>, there are two variants..."

Reply:
Changed to "There are two variants of the `reportEvent` API for event-level reporting"